src/app/ngx-mask/test/separator.spec.ts

```diff
@@ -295,6 +295,23 @@ describe('Separator: Mask', () => {
         expect(inputTarget.selectionStart).toEqual(0);
     });
 
+    // it('not change dot with comma with backspce on last digits', () => {
```

Comment:
@enricooliva  Why did you comment tests ?

Reply:
I'm sorry @Thegrep01 I tried to run it but the inputTarget.value not change after the triggerEventHandler, I will check again.